docs/content/v0.20/architecture.md

```diff
@@ -229,20 +225,39 @@ pattern has been extended to provide the following features:
     into/from the OCM mailbox namespaces.
 
 Currently there are two controllers in the KubeStellar controller
-manager: the placement controller and the status controllor.
+manager: the binding controller and the status controllor.
+
+### Planned changes to the above:
+
+as part of the decoupling of transport from binding as described above, the following changes are included:
+- Binding Controller - one client connected to the WDS space and 
+  (or more in the future) to connect to one or more ITS shards.
```

Comment:
The "(or more in the future)" is not properly attached to anything, I do not know what it is modifying.

Reply:
fixed